connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -106,6 +116,12 @@ public void onStartup(String connector) {
     }
 
     @Override
+    public void onPause(String connector) {
```

Comment:
Why do tasks have `onStartup` and `onResume` which both set the status to `RUNNING`, but connectors only have `onStartup`?

Looking at `WorkerConnector`/`WorkerTask` it seems like we just handle some naming sort of inconsistently? The `transitionTo` methods are implemented differently but I'm not sure why.